vnext/Microsoft.ReactNative/ReactHost/ReactContext.h

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "React.h"
+
+namespace Mso::React {
+
+class ReactInstanceWin;
+
+class ReactContext final : public Mso::UnknownObject<IReactContext> {
+ public:
+  ReactContext(
+      Mso::WeakPtr<ReactInstanceWin> &&reactInstance,
+      winrt::Microsoft::ReactNative::IReactPropertyBag const &properties,
+      winrt::Microsoft::ReactNative::IReactNotificationService const &notifications) noexcept;
+
+  // ReactContext may have longer lifespan than ReactInstance.
```

Comment:
>may have longer [](start = 18, length = 15)

(nit) is this because components may hold on to ReactContext object?

Should there be Api IsDestroyed? Or we just crash for other public methods when this object is destroyed is ok?

Reply:
I do not know - I simply moved this code out of ReactInstanceWin.(h|cpp) into it's own .h/.cpp - I'm not planning to alter how it works or what it does with this change.